x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table_header.tsx

```diff
@@ -34,6 +43,16 @@ interface ButtonProps {
   onClick(): void;
 }
 
+const LastModificationLabel = ({ lastModified }: { lastModified: string }) => (
+  <EuiText size="s">
+    <FormattedMessage
+      id="xpack.csp.rules.lastModificationLabel"
```

Comment:
try to use more sub sections eg: `rules.rulesTable.rulesTableInfo.lastModificationLabel`